packages/ts/src/components/bullet-legend/constants.ts

```diff
@@ -0,0 +1,2 @@
+// Default bullet size in pixels (used as fallback when bulletSize is null)
+export const DEFAULT_BULLET_SIZE = '20'
```

Comment:
Is there a reason to provide it as a string here? How does it correspond to the `--vis-legend-bullet-size` CSS variable?

Reply:
Great point! Will update that to be a number.

Reply:
Done.